charts/web3signer-validators/templates/statefulset.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.enabled }}
 {{- $root := . -}}
 {{- $counter := 0 }}
-{{- range (untilStep 0 (int .Values.validatorsCount) 1) }}
+{{- range (untilStep (int .Values.validatorsKeyIndex) (add .Values.validatorsKeyIndex .Values.validatorsCount) 1) }}
```

Comment:
Hey @dominikkaminski, definitely like this, thanks for the PR. Are you sure this is correct? Getting this error:

```
Error: template: web3signer-validators/templates/statefulset.yaml:4:93: executing "web3signer-validators/templates/statefulset.yaml" at <.Values.validatorsCount>: wrong type for value; expected int; got int64
helm.go:84: [debug] template: web3signer-validators/templates/statefulset.yaml:4:93: executing "web3signer-validators/templates/statefulset.yaml" at <.Values.validatorsCount>: wrong type for value; expected int; got int64
```

Using these values:

```
validatorsCount: 5
validatorsKeyIndex: 5
```

Reply:
Hey @unxnn this time you were faster in testing as I was :pensive: 

Now is tested and the issue is fixed (i didnt expect that untilStep could not handle the return of int64 from add)